app/controllers/map_controller.rb

```diff
@@ -1,22 +1,20 @@
 class MapController < ApplicationController
   def index
     @title = 'Maps'
-    @pagy, @nodes = pagy(Node.order('nid DESC')
-      .where(type: 'map', status: 1), items: 32)
+    nodes = Node.find_by_tag('map-archive')
+      .order('nodes.nid DESC')
```

Comment:
This should be ```node.nid DESC```, since that is the name of the column.

```suggestion
      .order('node.nid DESC')
```